src/ClisBnbStrategy.sol

```diff
@@ -0,0 +1,339 @@
+            SafeERC20.safeIncreaseAllowance(IERC20(asset_), address(strategyStorage.slisBnbProvider), assets);
+            // provide the slisBnb to the slisBnb provider contract and yieldnest's institutional wallet set to recipient of clisBnb received
+            ISlisBnbProvider(strategyStorage.slisBnbProvider).provide(assets, strategyStorage.yieldNestMpcWallet);
+        }
+    }
+
+    /**
+     * @notice Stakes slisBnb tokens into the provider contract
+     * @dev Only callable by accounts with KEEPER_ROLE. This will be used if there is additional slisBnb
+     * delegated to this strategy when sync deposit is disabled and for rewards sent to this strategy
+     * @param amount The amount of slisBnb tokens to stake
+     */
+    function stakeSlisBnb(uint256 amount) external onlyRole(KEEPER_ROLE) {
+        address slisBnb = getSlisBnb();
+        StrategyStorage storage strategyStorage = _getStrategyStorage();
+        // increase allowance for the slisBnb provider contract
+        SafeERC20.safeIncreaseAllowance(IERC20(slisBnb), address(strategyStorage.slisBnbProvider), amount);
+        // provide the slisBnb to the slisBnb provider contract and yieldnest's institutional wallet set to recipient of clisBnb received
+        ISlisBnbProvider(strategyStorage.slisBnbProvider).provide(amount, strategyStorage.yieldNestMpcWallet);
+    }
+
+    /**
+     * @notice Internal function to handle withdrawals for base asset(i.e. slisBnb for this strategy).
+     * @param asset_ The address of the asset.
+     * @param caller The address of the caller.
+     * @param receiver The address of the receiver.
+     * @param owner The address of the owner.
+     * @param assets The amount of assets to withdraw.
+     * @param shares The equivalent amount of shares.
+     */
+    function _withdrawAsset(
+        address asset_,
+        address caller,
+        address receiver,
+        address owner,
+        uint256 assets,
+        uint256 shares
+    ) internal virtual override {
+        // check if the asset is withdrawable
+        if (!_getBaseStrategyStorage().isAssetWithdrawable[asset_]) {
+            revert AssetNotWithdrawable();
+        }
+        // withdraw is allowed only for the base asset(i.e. slisBnb for this strategy)
+        if (asset_ != asset()) {
+            revert UnSupportedAsset(asset_);
+        }
+
+        // call the base strategy withdraw function for accounting
+        _subTotalAssets(assets);
+
+        if (caller != owner) {
+            _spendAllowance(owner, caller, shares);
+        }
+
+        // NOTE: burn shares before withdrawing the assets
+        _burn(owner, shares);
+
+        uint256 vaultBalance = IERC20(asset_).balanceOf(address(this));
+
+        StrategyStorage storage strategyStorage = _getStrategyStorage();
+        // if the vault balance is less than the assets to withdraw, unstake the slisBnb
+        if (vaultBalance < assets) {
```

Comment:
again this assumes what's withdrawn is slisBNB

so this assumes asset_ == slisBNB

let's check that in the if, looks like that would fix it

Additionally feels, like syncDeposit boolean can apply here

Reply:
Ok i see the check

```        // withdraw is allowed only for the base asset(i.e. slisBnb for this strategy)
        if (asset_ != asset()) {
            revert UnSupportedAsset(asset_);
        }
```

that looks fine. 

Adding syncWtihdraw indeed doesn't seem to have a ton of benefit, it would prevent unstaking but that is desired now